src/index.js

```diff
@@ -45,3 +45,22 @@ exports.Queue = Plotly.Queue;
 
 // export d3 used in the bundle
 exports.d3 = require('d3');
+
+Plotly.register({
+    traces: {
+        bar: require('./traces/bar'),
```

Comment:
agreed start simple. Might be worth ensuring there is an upgrade path to more complex notation (that will still support the simpler API) in case our needs change down the line.